src/api.coffee

```diff
@@ -45,6 +45,10 @@ IS_LOCAL = window.location.port is '8000' or window.__karma__
 # Make sure API calls occur **after** all local Action listeners complete
 delay = (ms, fn) -> setTimeout(fn, ms)
 
+# Join an array
+arrayToParams = (vals, name) ->
+  _.map(vals, (value) -> "#{name}[]=#{value}").join('&')
```

Comment:
could we either use [qs](https://www.npmjs.com/package/qs) or jQuery's [`.param`](http://api.jquery.com/jquery.param/) instead?